proto/asgt/type/model.proto

```diff
@@ -23,4 +24,5 @@ message Model {
   InputType input_type = 8;
   string dataset_type = 9;
   repeated asgt.type.ModelFile model_files = 10 [(gen_bq_schema.bigquery).ignore = true];
+  repeated asgt.type.Confidence confidence_thresholds = 11 [(gen_bq_schema.bigquery).ignore = true];
```

Comment:
don't you think it's enough to do `repeated float confidence_thresholds = 11` ?

Reply:
i like pseudo KV 👍  It is more future/stupid proof